common/testData/localeIdentifiers/localeCanonicalization.txt

```diff
@@ -1153,6 +1221,7 @@ aaa_Adlm_AC_arevela_fonipa	;	aaa_Adlm_AC_fonipa
 aaa_Adlm_AC_arevmda_fonipa	;	aaa_Adlm_AC_fonipa
 aaa_Adlm_AC_bokmal_fonipa	;	aaa_Adlm_AC_fonipa
 aaa_Adlm_AC_fonipa_hakka	;	aaa_Adlm_AC_fonipa
+aaa_Adlm_AC_fonipa_hepburn_heploc	;	aaa_Adlm_AC_alalc97_fonipa
```

Comment:
Should this canonicalise to aaa-Adlm-AC-alalc97-fonipa-hepburn ?

1. Alias "heploc" => "alalc97"
2. No aliases for "fonipa" or "hepburn"
3. Variant sort order is lexical: "alalc97-fonipa-hepburn"